ui/app/AppLayouts/Chat/ChatColumn/SuggestionFilter.qml

```diff
@@ -32,9 +32,16 @@ Item {
         if (!isFilteringPropertyOk())
             return
 
-        var length = sourceModel.count
+        var length = sourceModel.rowCount()
         for (var i = 0; i < length; ++i) {
-            var item = sourceModel.get(i);
+            const publicKey = sourceModel.rowData(i, "publicKey");
+            var item = {
+                alias: sourceModel.rowData(i, "alias"),
+                userName: sourceModel.rowData(i, "userName"),
+                publicKey: publicKey,
+                identicon: getProfileImage(publicKey, false, false) || sourceModel.rowData(i, "identicon"),
+                localName: sourceModel.rowData(i, "localName")
+            }
```

Comment:
I would wish we could use the sourceModel as is, since all the keys are matching. Even getProfileImage may probably be moved to nim... You certainly know better.

Reply:
I tried exactly what you suggest but couldn't make it work. ListModel have `.get(i: index)` method but I don't think this is available to us in our nim implementation. If you know how to make it work, I'd appreciate advice